src/sentry/static/sentry/app/views/settings/components/settingsBreadcrumb/index.jsx

```diff
@@ -21,22 +26,29 @@ const MENUS = {
 class SettingsBreadcrumb extends React.Component {
   static propTypes = {
     routes: PropTypes.array,
+    pathMap: PropTypes.object,
   };
 
   static contextTypes = {
     organization: SentryTypes.Organization,
   };
 
+  componentDidUpdate(prevProps) {
+    if (this.props.routes === prevProps.routes) return;
+    SettingsBreadcrumbActions.trimMappings(this.props.routes);
+  }
+
   render() {
-    let {routes, params} = this.props;
-    let routesWithNames = routes.filter(({name}) => name);
-    let lastRouteIndex = routesWithNames.length - 1;
+    let {routes, params, pathMap = {}} = this.props;
```

Comment:
We should use react's `defaultProps` for `pathMap`

Reply:
f6b1cf3